core/src/test/java/io/questdb/griffin/AlterTableCommitLagTest.java

```diff
@@ -239,85 +267,102 @@ public boolean rename(LPSZ from, LPSZ to) {
     }
 
     @Test
-    public void setMaxUncommittedRowsFailsToReopenBackMetaFile() throws Exception {
+    public void setMaxUncommittedRowsMissingEquals() throws Exception {
+        assertFailure("ALTER TABLE X SET PARAM maxUncommittedRows 100",
+                "CREATE TABLE X (ts TIMESTAMP, i INT, l LONG) timestamp(ts) PARTITION BY MONTH",
+                43,
+                "'=' expected");
+    }
+
+    @Test
+    public void setMaxUncommittedRowsNegativeValue() throws Exception {
+        assertFailure("ALTER TABLE X SET PARAM maxUncommittedRows = -1",
+                "CREATE TABLE X (ts TIMESTAMP, i INT, l LONG) timestamp(ts) PARTITION BY MONTH",
+                24,
+                "invalid value [value=-,parameter=maxUncommittedRows]");
+    }
+
+    @Test
+    public void setUnknownParameter() throws Exception {
         assertMemoryLeak(() -> {
             try (TableModel tbl = new TableModel(configuration, "X", PartitionBy.DAY)) {
                 createX(tbl);
             }
-            engine.releaseAllWriters();
-            ff = new FilesFacadeImpl() {
-                int attempt = 0;
-
-                @Override
-                public long openRO(LPSZ path) {
-                    if (Chars.endsWith(path, TableUtils.META_FILE_NAME) && attempt++ == 1) {
-                        return -1;
-                    }
-                    return super.openRO(path);
+            try (TableReader ignored = engine.getReader(AllowAllCairoSecurityContext.INSTANCE, "X")) {
+                try {
+                    compiler.compile("ALTER TABLE X SET PARAM vommitLag = 111s", sqlExecutionContext);
```

Comment:
typo

Reply:
this typo is intentional, we assert that incorrect parameter name is handled gracefully :)